lib/ClassUtils.lua

```diff
@@ -24,11 +24,15 @@ function ClassUtils.makeClass(name, constructor)
     return Class
 end
 
-function ClassUtils.makeWrapperClass(name)
+function ClassUtils.makeConstructedClass(name, constructor)
```

Comment:
Aren't these changes unrelated to the async stuff?

Reply:
Yes, please can we merge with this change to fix tests on roblox as that's dependent on this branch, and I'll apply the proposed change immediately after